components/dependency_injection/advanced.rst

```diff
@@ -219,6 +219,79 @@ You can change the inner service name if you want to:
             ->setPublic(false)
             ->setDecoratedService('foo', 'bar.wooz');
 
+.. versionadded:: 2.8
+    The ability to define the decoration priority was introduced in Symfony 2.8.
+    Prior to Symfony 2.8, the priority depends on the order in
+    which definitions are found.
+
+If you want to apply more than one decorator to a service, you can control their
+order by configuring the priority of decoration. Decorators with higher
```

Comment:
I'll add a very brief mention about the allowed values for the `decoration_priority`: positive integers only? positive and negative integers?